src/selectors/wiki.js

```diff
@@ -6,8 +6,10 @@ export const getWikiOptions = createSelector(
 	( wikis = new Map() ) => {
 		return wikis.map( ( wiki ) => ( {
 			value: wiki.id,
-			label: `${wiki.name} (${wiki.domain})`
-		} ) ).toArray();
+			label: `${wiki.domain}`
+		} ) )
+			.toArray()
+			.sort( ( a, b ) => a.label > b.label );
```

Comment:
@dayllanmaza you should perform the sort before the conversion to an array since Immutable's sort is probably faster than JavaScripts. :)

Reply:
@dayllanmaza actually... you should sort in the reducer (when the wikis are being added) that way it will only be sorted once. :)

Reply:
Done